package/obs/rmt-server.changes

```diff
@@ -1,3 +1,14 @@
+-------------------------------------------------------------------
+Thu Jul  9 17:39:04 UTC 2020 - Luís Caparroz <luis.caparroz@suse.com>
+
+- Version 2.5.11
+- Update the Ruby gem `responders` to version 3.0.1: the previous version on the
+  project was using the gem 'rack' version 2.1.4, which had a security
+  vulnerability reported on CVE-2020-8184.
+- Upgrade Rails from version 5.1.6.2 to 5.2.4.3, due to several vulnerabilities
+  which were fixed in the later version
+  (https://github.com/rails/rails/releases/tag/v5.2.4.3).
```

Comment:
This level of detail isn't really required for the changelogs, which version of Rails is under the hood, 5.1.6.2 or 5.2.4.3, is of no consequence to RMT users.

CVE numbers however are useful, ideally all of the CVEs need to be collected. So a short message like `Security update of the dependencies (bsc#XXXXX, CVE-YYYY-YYYY, CVE-ZZZZ-ZZZZ)` would be much more preferable IMO.

Reply:
I updated the changelog in 07b75dd.

Reply:
:+1: